src/main/java/com/derongan/minecraft/mineinabyss/player/PlayerExtensions.kt

```diff
@@ -0,0 +1,5 @@
+package com.derongan.minecraft.mineinabyss.player
+
+import org.bukkit.entity.Player
+
+fun Player.openHubStorage() = this.openInventory(this.enderChest)
```

Comment:
Im concerned about using the enderChest in all cases. Maybe in a future change we can instead use a custom storage container.

Reply:
@Derongan I think this works ok for now, could you re-review and merge if you approve?